app/models/status.rb

```diff
@@ -90,6 +90,7 @@ class Status < ApplicationRecord
   scope :including_silenced_accounts, -> { left_outer_joins(:account).where.not(accounts: { silenced_at: nil }) }
   scope :not_excluded_by_account, ->(account) { where.not(account_id: account.excluded_from_timeline_account_ids) }
   scope :not_domain_blocked_by_account, ->(account) { account.excluded_from_timeline_domains.blank? ? left_outer_joins(:account) : left_outer_joins(:account).where('accounts.domain IS NULL OR accounts.domain NOT IN (?)', account.excluded_from_timeline_domains) }
+  scope :not_from_bot, -> { left_outer_joins(:account).where.not('accounts.actor_type': %w(Application Service)) }
```

Comment:
When an account is created by using command line:
```
bin/tootctl accounts create normal --email=normal@localhost --confirmed
```
The `actor_type` will be `[null]` is stead of `Person`, `Service` or `Application`.

Reply:
You probably want something like the following:
```
{ left_outer_joins(:account).where('accounts.actor_type IS NULL OR accounts.actor_type NOT IN (?)', %w(Application Service)) }
```

Reply:
Update:
Tried using the line that I provided here. It fixed "Local timeline and community timeline" bug that I reported. Home timeline bug remains unfixed.